### DISCUSSION TITLE
Edit form submission via CP

### DISCUSSION THREAD

**Discussion by beatwiz** [statamic/cms#7221] [Q&A] [answered]
Hi there and thanks in advance!

Is there any way to allow form submissions to be edited via CP? Currently all submissions show as "Read only".

Best,

**Comment by jasonvarga** [comment, marked as answer]
Not at the moment, without building something custom.

Another option could be to use https://statamic.com/addons/duncanmcclean/guest-entries to create entries instead of form submissions. Then you could edit entries as normal in the CP.

**Reply by beatwiz** [reply]
Thanks! I'll do that :)
